pom.xml

```diff
@@ -23,7 +23,7 @@
         <alfresco.azure-connector.version>5.0.3</alfresco.azure-connector.version>
         <alfresco.salesforce-connector.version>3.2.0</alfresco.salesforce-connector.version>
         <alfresco.outlook.version>2.9.0</alfresco.outlook.version>
-        <alfresco.desktop-sync.version>5.1.0</alfresco.desktop-sync.version>
+        <alfresco.desktop-sync.version>5.2.0-A1</alfresco.desktop-sync.version>
```

Comment:
Hi @cezary-witkowski, The following change needs to be performed in 2 more files. please refer to this https://github.com/Alfresco/acs-packaging/commit/6993b5318c6d7ca7440aa830850430296f4fcaf5.